src/code42cli/cmds/devices.py

```diff
@@ -145,13 +159,16 @@ def _get_device_info(sdk, device_guid):
 
 active_option = click.option(
     "--active",
-    required=False,
-    type=bool,
     is_flag=True,
+    help="Limits results to only active devices.",
     default=None,
-    help="Get only active or deactivated devices. Defaults to getting all devices.",
 )
-
+inactive_option = click.option(
+    "--inactive",
+    is_flag=True,
+    help="Limits results to only deactivated devices.",
+    cls=incompatible_with("active"),
+)
```

Comment:
Since the `active` param actually takes 3 states (None, True, False) the boolean option type didn't work.

Reply:
I'm not in favor of dropping `--drop-most-recent`.  The use case is that many customers want to be sure that a user always has at least one active device.  There is no way to know the "right" date to pick.  This parameter should stay.